xapian-letor/include/xapian-letor/scorer.h

```diff
@@ -68,6 +68,19 @@ class XAPIAN_VISIBILITY_DEFAULT NDCGScore: public Scorer {
 
 };
 
+/// ERRScore class
+class XAPIAN_VISIBILITY_DEFAULT ERRScore: public Scorer {
+  public:
+    /// Default constructor
+    ERRScore();
+
+    /// Destructor
+    ~ERRScore();
+
+    double score(const std::vector<FeatureVector> & fvv) const;
+
```

Comment:
We didn't catch this on the previous scorer (since it's been there for ages), but we prefer not to have blank lines at the end of class declarations. It's probably easiest to tidy this up when we do a pass using `clang-format`, but if you wanted to tidy it up quickly as a separate commit in this PR I'd be fine with that. (It's not a fixup commit, though.)

Reply:
This should be a separate commit so that I can eliminate this problem with the other scorer as well??